products/urls.py

```diff
@@ -1,4 +1,7 @@
-from django.urls import path
+from django.urls    import path
+
+from products.views import DetailView
 
 urlpatterns = [
+    path('/detail/<int:product_id>', DetailView.as_view()),
```

Comment:
만약 1번 product의 상세 정보를 전달하는 endpoint라면 `:8000/products/1` 정도로도 충분할 것 같습니다.
```suggestion
    path('/<int:product_id>', DetailView.as_view()),
```